uvplot/io.py

```diff
@@ -102,6 +113,8 @@ def export_uvtable(uvtable_filename, tb, vis="", split_args=None, split=None,
                     (MStb_name, split_args['vis']))
 
         split(**split_args)
+        # after splitting data is put into "DATA" column
```

Comment:
Is this always true? I haven't checked.

Reply:
I do believe so. From page 267 of the CASA cookbook, detailing the split function: 

"Usually you will run split with datacolumn=’corrected’ as previous operations (e.g. applycal) will have placed the calibrated data in the CORRECTED DATA column of the MS. This will produce
a new MS with this corrected data in its DATA column."

And there are no inputs that seem to allow you to change what column the data gets outputted to.

Reply:
Ok, thank you for checking.